src/app/variant-reports/variant-reports.ts

```diff
@@ -203,7 +204,7 @@ export class VariantReport {
 
 export class PedigreeTable {
   public constructor(
-    public readonly pedigrees: PedigreeCounter[][],
+    public pedigrees: PedigreeCounter[][],
```

Comment:
pedigrees cannot be readonly because it is changed
variant-reports.component.ts:112:13
Cannot assign to 'pedigrees' because it is a read-only property.

112       table.pedigrees = this.chunkPedigrees(newCounters[table.groupName]);